tests/drivers/fprotect/negative/testcase.yaml

```diff
@@ -1,25 +1,5 @@
 tests:
   drivers.fprotect.negative:
-    platform_allow: nrf5340pdk_nrf5340_cpuapp nrf5340pdk_nrf5340_cpuappns nrf9160dk_nrf9160 nrf52840dk_nrf52840 nrf52dk_nrf52832
-      nrf5340dk_nrf5340_cpuapp nrf5340dk_nrf5340_cpuappns
-    tags: b0 fprotect
-    harness: console
-    harness_config:
-      type: multi_line
-      ordered: true
-      regex:
-        - "NOTE: A BUS FAULT immediately after this message means the test passed!"
-        - "E: \\*\\*\\*\\*\\* BUS FAULT \\*\\*\\*\\*\\*"
-        - "E:   (Imprecise|Precise) data bus error"
-        - "E: Halting system"
-  drivers.fprotect.negative.nrf51:
-    platform_allow: nrf51dk_nrf51422
-    tags: b0 fprotect
-    harness: console
-    harness_config:
-      type: multi_line
-      ordered: true
-      regex:
-        - "NOTE: A BUS FAULT immediately after this message means the test passed!"
-        - "E: \\*\\*\\*\\*\\* HARD FAULT \\*\\*\\*\\*\\*"
-        - "E: Halting system"
+    platform_allow: nrf5340pdk_nrf5340_cpuapp nrf5340pdk_nrf5340_cpuappns nrf9160dk_nrf9160 nrf52dk_nrf52832
```

Comment:
as befare, we should remove pdk here since support for it will be removed

Reply:
Done